openfecli/parameters/mol.py

```diff
@@ -20,8 +20,19 @@ def _load_molecule_from_smiles(user_input, context):
     return Molecule(rdkit=mol)
 
 
+def _load_molecule_from_sdf(user_input, context):
+    from openfe.setup import Molecule
+    try:
+        with open(user_input, mode="r") as sdf:
+            contents = sdf.read()
+        return Molecule.from_sdf_string(contents)
```

Comment:
I'm not really a fan of blindly reading an entire file.  Would it be possible to add `Molecule.from_sdf_file` where inside this creates a `SDMolSupplier` with the path

Reply:
Done. Take another look now that tests pass?